PraceDomowe/PracaDomowa5/praca_domowa_5.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<p>Prawie dobrze, jedyny błąd polega na tym że liczba pytań w zbiorze to nie są tylko te Posts dla których istnieje AcceptedAnswer, tylko te, których PostTypeId = 1 (chociaż to trzeba by było odczytać z opisu danych).</p><p> </p><p><code>2 punkty</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>